src/ui/actions/timeline.ts

```diff
@@ -177,7 +177,7 @@ function onPaused({ time }: PauseEventArgs): UIThunkAction {
     dispatch(setTimelineState({ currentTime: time, playback: null }));
 
     try {
-      if (!isRepaintEnabled()) {
+      if (!isRepaintEnabled() || !isRegionLoaded(store.getState(), time)) {
```

Comment:
Having a comment here would be nice, since we're trying to strike a balance between showing graphics quickly vs. avoiding jitter due to showing the most recent paint and then replacing them with the repainted graphics soon afterwards.